lms/services/canvas_api/_basic.py

```diff
@@ -50,18 +50,34 @@ def send(
         params=None,
         headers=None,
         url_stub="/api/v1",
+        timeout=(10, 10),
```

Comment:
Hmm. I _think_ `BasicClient` might actually only ever be called by `AuthenticatedClient` which does pass the `timeout` arg, so possibly I can omit the default value here...